Doc/whatsnew/3.11.rst

```diff
@@ -70,6 +70,49 @@ Summary -- Release highlights
 New Features
 ============
 
+.. _whatsnew310-pep657:
+
+Enhanced error locations in tracebacks
```

Comment:
By the let's either make one of the existing or add a new example about multiple annotations 
```
Traceback (most recent call last):
  File "test.py", line 14, in <module>
    lel3(x)
    ^^^^^^^
  File "test.py", line 12, in lel3
    return lel2(x) / 23
           ^^^^^^^
  File "test.py", line 9, in lel2
    return 25 + lel(x) + lel(x)
                ^^^^^^
  File "test.py", line 6, in lel
    return 1 + foo(a,b,c=x['z']['x']['y']['z']['y'], d=e)
                         ^^^^^^^^^^^^^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable
```
(maybe with better naming

Reply:
Let me adopt that example with some different names.

Reply:
Changed to this example with different names.